corehq/apps/commtrack/tests/test_rebuild.py

```diff
@@ -116,6 +119,6 @@ def test_edit_submissions_simple(self):
             form_id=form_id,
         )
         case = CommCareCase.get(id=self.case.case_id)
-        self.assertEqual(1, len(case.actions))
+        self.assertEqual(2, len(case.actions))
```

Comment:
It's because of this:
```
 -       self.case = CommCareCase(domain=self.domain)
 -       self.case.save()
 +      self.case = CaseFactory(domain=self.domain).create_case()
```